v19.2/cockroachcloud-connect-to-your-cluster.md

```diff
@@ -185,269 +175,183 @@ For more details about the built-in SQL client, and many examples of how to use
+~~~ python
 
-On the machine where you want to run your application:
+# Create an engine to communicate with the database. The "cockroachdb://" prefix
+# for the engine URL indicates that we are connecting to CockroachDB.
+engine = create_engine('cockroachdb://<username>:<password>@<region>.<cluster_name>:26257/<database>?sslmode=verify-full&sslrootcert=<absolute path to CA certificate>')
 
-1. Install the SQLAlchemy ORM as well as a [CockroachDB Python package](https://github.com/cockroachdb/cockroachdb-python) that accounts for some minor differences between CockroachDB and PostgreSQL:
+~~~
 
-    {% include copy-clipboard.html %}
-    ~~~ shell
-    $ pip install sqlalchemy cockroachdb
-    ~~~
+</section>
+
+<section class="filter-content" markdown="1" data-scope="go">
+
+Start by choosing the [Go pq driver](https://godoc.org/github.com/lib/pq) or [GORM ORM](http://gorm.io/):
+
+- [Go pq driver](#go-pq-driver)
+- [GORM](#gorm)
+
+#### Go pq driver
+
+{% include copy-clipboard.html %}
+~~~ go
+  //Connect to the database.
+	db, err := sql.Open(
+		"postgres",
+		"postgresql://<username>:<password>@<region>.<cluster_name>:26257/<database>?sslmode=verify-full&sslrootcert=<absolute path to CA certificate>",
+	)
+	if err != nil {
+		log.Fatal("error connecting to the database: ", err)
+	}
+
+~~~
+
+#### GORM
+
+{% include copy-clipboard.html %}
+~~~ go
+
+  // Connect to the database.
+    const addr = "postgresql://<username>:<password>@<region>.<cluster_name>:26257/<database>?sslmode=verify-full&sslrootcert=<absolute path to CA certificate>"
+    db, err := gorm.Open("postgres", addr)
+    if err != nil {
+        log.Fatal(err)
+    }
+~~~
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="js">
+
+Start by choosing the [Node.js pg driver](https://www.npmjs.com/package/pg) or [Sequelize ORM](https://sequelize.readthedocs.io/en/v3/):
 
-    For other ways to install SQLAlchemy, see the [official documentation](http://docs.sqlalchemy.org/en/latest/intro.html#installation-guide).
+- [Node.js pg driver](#node-js-pg-driver)
+- [Sequelize ORM](#sequelize-orm)
 
-2. Run code to execute basic SQL statements.
+#### Node.js pg driver
 
-    In the following code, the [SQLAlchemy ORM](https://docs.sqlalchemy.org/en/latest/) maps Python-specific objects to SQL operations. Specifically, `Base.metadata.create_all(engine)` creates an `accounts` table based on the `Account` class, `session.add_all([Account(),...])` inserts rows into the table, and `session.query(Account)` selects from the table so that balances can be printed.
+{% include copy-clipboard.html %}
+~~~ js
 
-    {{site.data.alerts.callout_info}}
-    The [CockroachDB Python package](https://github.com/cockroachdb/cockroachdb-python) installed earlier is triggered by the `cockroachdb://` prefix in the engine URL. Using `postgres://` to connect to your cluster will not work.
-    {{site.data.alerts.end}}
+// Connect to the "bank" database.
+var config = {
```

Comment:
I had to add `password` to the config to get it to connect.

Reply:
Yep. Good catch. Fixed.